### PR TITLE
source-mention contribution from ombhardwajj

### LINKED ISSUE #46014: [connector-request] Mention
This is a **suggestion** to create a connector for **Mention** using the UI Connector Builder.
The website for the connector is https://mention.com.

If you want to try, claim this issue and start working on it following the steps below.

### Steps:
1. Comment in the issue and wait to be assigned to start working on it.
2. Map the API endpoints (get approval to move to 3 step)
3. Describe steps to get credentials.
4. Create the connector using the UI Builder

> [!IMPORTANT]
> Airbyte doesn't have sandbox credentials for this connector. You must have access/credentials to the service provider to create the connector.

> [!CAUTION]
> This is a **suggestion** and there may be cases where creating the connector using the Builder won't be possible. To minimize the risk of investing a lot of time directly in creating the connector, we strongly recommend to follow steps above.

### PR DESCRIPTION
**Pull request by ombhardwajj**
This closes #46014 
## What

This PR adds a new connector for source Mention (source-mention).

Contributor-provided connector description:

Mention is a Social Listening and Media Monitoring Tool.
This connector allows you to extract data from various Mention APIs such as Accounts , Alerts , Mentions , Statistics and others
Docs: https://dev.mention.com/current/
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `api_key` | `string` | API Key.  |  |
| `from` | `string` | from. stats are to be fetched for mentions retrieved after that date |  |
| `to` | `string` | to. stats are fetched for mentions retrieved before that date |  |
| `interval` | `string` | interval. Periodicity of statistics returned. it may be daily(P1D), weekly(P1W) or monthly(P1M). | P1D |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| Alerts | id | DefaultPaginator | ✅ |  ❌  |
| Alert | id | No pagination | ✅ |  ❌  |
| Mentions | id | DefaultPaginator | ✅ |  ❌  |
| Mention | id | No pagination | ✅ |  ❌  |
| Mention Children |  | DefaultPaginator | ✅ |  ❌  |
| Accounts | id | No pagination | ✅ |  ❌  |
| Account | id | No pagination | ✅ |  ❌  |
| Tags |  | No pagination | ✅ |  ❌  |
| Alert Authors |  | DefaultPaginator | ✅ |  ❌  |
| Alert Tasks |  | DefaultPaginator | ✅ |  ❌  |
| Statistics |  | No pagination | ✅ |  ❌  |
| Tasks | id | No pagination | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

**Review by natikgadzhi — DISMISSED** [review]
@marcosmarxm I think you should set the expectation with folks that PRs without an icon just won't be reviewed at all.

Ping me, I can craft a CI check that verifies that the icon is not the default one.

**Comment by ombhardwajj** [comment]
@natikgadzhi I've added the icon 🙇

File: airbyte-integrations/connectors/source-mention/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,11358 @@
...
+      name: Alerts
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: >-
+            https://api.mention.net/api/accounts/{{ stream_partition.account_id
+            }}/alerts/
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mention/manifest.yaml:31]: Fix this path

File: airbyte-integrations/connectors/source-mention/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,11358 @@
...
+        title: from
+      to:
+        type: string
+        description: stats are fetched for mentions retrieved before that date
+        order: 2
+        title: to
+      interval:
+        type: string
+        description: >-
+          Periodicity of statistics returned. it may be daily(P1D), weekly(P1W)
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mention/manifest.yaml:570]: It looks from and to are required to get stats. Can you raise a discussion in connector-dev Slack channel about what is the best implementation in this case?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Couple of comments.

File: airbyte-integrations/connectors/source-mention/manifest.yaml (hunk 2 above)
Comment (ombhardwajj) [reply airbyte-integrations/connectors/source-mention/manifest.yaml:570]: Okay, I've sent a message there!

**Comment by marcosmarxm** [comment]
Hello 👋 Due to the numerous contributions happening, I decided to create this [view](https://github.com/orgs/airbytehq/projects/104/views/3) to help you know when I plan to review your contribution. If your connector has some limited-time credential feel free to reach me out.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**